app/browser/api/ledgerNotifications.js

```diff
@@ -199,11 +225,27 @@ const showEnabledNotifications = (state) => {
     } else if (shouldShowNotificationAddFunds()) {
       showAddFunds()
     }
-  } else if (reconcileStamp - new Date().getTime() < 2 * ledgerUtil.milliseconds.day) {
+  } else if (reconcileStamp - now < 2 * ledgerUtil.milliseconds.day) {
     if (sufficientBalanceToReconcile(state) && (shouldShowNotificationReviewPublishers())) {
-      showReviewPublishers(new Date().getTime() + ledgerUtil.milliseconds.day)
+      showReviewPublishers(now + ledgerUtil.milliseconds.day)
     }
+  } else if (now - bootStamp > backupNotifyTimestamp) {
+    if (!aboutPreferencesState.hasBeenBackedUp(state) && hasFunded(state)) {
+      const nextTime = now + getNextBackupNotification(state, backupNotifyCount + 1, backupNotifyInterval) // set next time to notify case for remind later
+      state = aboutPreferencesState.setPreferencesProp(state, 'backupNotifyCount', backupNotifyCount + 1)
+      state = aboutPreferencesState.setPreferencesProp(state, 'backupNotifyTimestamp', nextTime)
+      module.exports.showBackupKeys(nextTime)
+    }
+  }
+
+  return state
+}
+
+const getNextBackupNotification = (state, count, interval) => {
```

Comment:
please add check if `interval` exists and if it's an array

Reply:
Done. WIP